data_collection/gazette/spiders/mg_belohorizonte.py

```diff
@@ -0,0 +1,51 @@
+from datetime import date, datetime, timedelta
+
+from scrapy import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class MgBeloHorizonteSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3106200"
+    name = "mg_belo_horizonte"
+    allowed_domains = ["portal6.pbh.gov.br"]
+    documents_url = (
+        "http://portal6.pbh.gov.br/dom/iniciaEdicao.do"
+        "?method=DomDia&dia={}&comboAno={}"
+    )
+    start_date = date(2002, 3, 21)
+
+    def start_requests(self):
+        current_date = date.today()
+        while current_date >= self.start_date:
+            formatted_date = current_date.strftime("%d/%m/%Y")
+            yield Request(
```

Comment:
Configurar a data de início para 01/08/2021 apenas um diário é retornado. Checando manualmente no site https://dom-web.pbh.gov.br/ existem mais de 4 edições entre 01 e 11 de Agosto de 2021.

Reply:
É, algo parece estranho. Rodando desde o início, o ano mais antigo é 2014. Foram coletados 1163 diários, o que dá uma média de pouco mais de um diário por ano - incompatível com o tamanho da cidade.